### PR TITLE
[IOTDB-3414] [mpp] fix schema query limit offset bug

### PR DESCRIPTION
**Pull request by xinzhongtianxia**

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/FakePartitionFetcherImpl.java [hunk 1]
```diff
@@ -43,7 +43,58 @@ public class FakePartitionFetcherImpl implements IPartitionFetcher {
...
+                new TDataNodeLocation()
+                    .setDataNodeId(12)
+                    .setExternalEndPoint(new TEndPoint("192.0.1.2", 9000))));
+    regionMap.put(new TSeriesPartitionSlot(device3.length()), region3);
+
+    schemaPartitionMap.put("root.sg", regionMap);
+
+    schemaPartition.setSchemaPartitionMap(schemaPartitionMap);
+
+    return schemaPartition;
```
Comment (SpriCoder) [inline server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/FakePartitionFetcherImpl.java:97]: Why add implementation in FakePartitionFetcherImpl?

Comment (xinzhongtianxia) [reply server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/FakePartitionFetcherImpl.java:97]: see `org.apache.iotdb.db.mpp.plan.plan.LogicalPlannerTest#parseSQLToPlanNode`, which passed a 
`FakePartitionFetcherImpl` to `Analyzer`.   But, the `FakePartitionFetcherImpl#getSchemaPartition` will return null by default, which cannot simulate multi-region scenarios. So I added the implementation to return some fake regions.

Comment (SpriCoder) [reply server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/FakePartitionFetcherImpl.java:97]: Get it

**Review by SpriCoder — APPROVED** [review]
LGTM！

**Comment by xinzhongtianxia** [comment]
Could you help merge this PR ? @xingtanzjr

**Pull request merged**